app/fastlane/helpers/android.rb

```
@@ -0,0 +1,92 @@
+
+      # Verify that the current version code is greater than the latest version on Play Store
+      # This method compares the versionCode in the gradle file against the latest version
+      # published on the Play Store internal track to ensure no version conflicts occur
+      #
+      # @param gradle_file [String] Path to the build.gradle file containing versionCode
+      # @return [void] Reports success or error based on version comparison
+      def android_verify_version_code(gradle_file)
+        latest = Fastlane::Actions::GooglePlayTrackVersionCodesAction.run(
+          track: "internal",
+          json_key: ENV["ANDROID_PLAY_STORE_JSON_KEY_PATH"],
+          package_name: ENV["ANDROID_PACKAGE_NAME"],
```

RuboCop (1.76.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)